registry/coder-labs/modules/copilot-cli/README.md

```diff
@@ -0,0 +1,213 @@
+  mutable     = true
+}
+
+module "copilot_cli" {
+  source   = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version  = "0.1.0"
+  agent_id = coder_agent.example.id
+  workdir  = "/home/coder/project"
+
+  github_token    = "your_github_token_here" # Or use data.coder_external_auth.github.access_token
+  ai_prompt       = data.coder_parameter.ai_prompt.value
+  copilot_model   = "claude-sonnet-4.5"
+  allow_all_tools = true
+  resume_session  = true
+
+  trusted_directories = ["/home/coder", "/tmp"]
+}
+```
+
+### Advanced Configuration with Specific Tool Permissions
+
+For more controlled environments where you want to specify exact tools:
+
+```tf
+module "copilot_cli" {
+  source   = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version  = "0.1.0"
+  agent_id = coder_agent.example.id
+  workdir  = "/home/coder/project"
+
+  allow_tools         = ["shell(git)", "shell(npm)", "write"]
+  trusted_directories = ["/home/coder/workspace", "/tmp"]
+
+  mcp_config = jsonencode({
+    mcpServers = {
+      filesystem = {
+        command     = "npx"
+        args        = ["-y", "@modelcontextprotocol/server-filesystem", "/home/coder/workspace"]
+        description = "Provides file system access to the workspace"
+        name        = "Filesystem"
+        timeout     = 3000
+        type        = "local"
+        tools       = ["*"]
+        trust       = true
+      }
+      playwright = {
+        command     = "npx"
+        args        = ["-y", "@playwright/mcp@latest", "--headless", "--isolated"]
+        description = "Browser automation for testing and previewing changes"
+        name        = "Playwright"
+        timeout     = 5000
+        type        = "local"
+        tools       = ["*"]
+        trust       = false
+      }
+    }
+  })
+
+  pre_install_script = <<-EOT
+    #!/bin/bash
+    curl -fsSL https://deb.nodesource.com/setup_22.x | sudo -E bash -
+    sudo apt-get install -y nodejs
+  EOT
+}
+```
+
+> [!NOTE]
+> GitHub Copilot CLI does not automatically install MCP servers. You have two options:
+>
+> - Use `npx -y` in the MCP config (shown above) to auto-install on each run
+> - Pre-install MCP servers in `pre_install_script` for faster startup (e.g., `npm install -g @modelcontextprotocol/server-filesystem`)
+
+### Direct Token Authentication
+
+```tf
+module "copilot_cli" {
+  source       = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version      = "0.1.0"
+  agent_id     = coder_agent.example.id
+  workdir      = "/home/coder/project"
+  github_token = "your_github_token_here" # Or use data.coder_external_auth.github.access_token
+}
+```
+
+### Standalone Mode
+
+Run and configure Copilot CLI as a standalone tool in your workspace.
+
+```tf
+module "copilot_cli" {
+  source       = "registry.coder.com/coder-labs/copilot-cli/coder"
+  version      = "0.1.0"
+  agent_id     = coder_agent.example.id
+  workdir      = "/home/coder"
+  report_tasks = false
+  cli_app      = true
+}
+```
```

Comment:
Does this mean it only gets installed and does not start?

Let's be clearer, **Standalone Mode** is not self-explanatory IMO.